### PR TITLE
Develop/student

### PR DESCRIPTION
**Pull request by banderan**

### CODE REVIEW

**Comment by banderan** [comment]
Finish, program works and mvn clean package shows success

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:6]: image

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+    private Color[] colors = Color.values();
+
     public String getRandomColor() {
-        return null;
+        int randomColor = random.nextInt(colors.length);
+        return colors[randomColor].toString();
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:11]: image

File: src/main/java/core/basesyntax/Number.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Number {
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Number.java:5]: Better delete this class and use Random in class Lottery

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Pull request closed without merging**